embed-code-go/test/embedding_instruction_test/embedding_instruction_test.go

```diff
@@ -97,48 +97,39 @@ func xmlAttribute(name string, value string) string {
 }
 
 func TestFalseXML(t *testing.T) {
```

Comment:
Who tf is "FalseXML"?

Is it (in fact) `EnsureXmlMisformed`, or something like that?